src/behavior.scm

```diff
@@ -295,7 +295,9 @@
 			(SequentialAnd
 				(Equal
 					(DefinedSchema "New departures")
-					(Get (State eye-contact-state (Variable "$x"))))
+					(Get
+						(TypedVariable (Variable "$x") (Type "NumberNode"))
+						(State eye-contact-state (Variable "$x"))))
```

Comment:
The above is valid, but should never actually be required.  The StateLink can only ever have one value, and if you set that value to something other than a NumberNode, the above will fail.   Basically, you are either hiding a bug, or moving a bug to a new location.